src/index.html

```diff
@@ -5,10 +5,109 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="navi">
+        <div class="navi__container">
+          <a
+            href="#home"
+            target="_blank"
+            class="navi__logoLink"
+          >
+            <img
+              src="./images/logo.png"
+              alt="logo"
+              class="navi__logoImg"
+            >
+          </a>
+          <nav class="navi">
```

Comment:
```suggestion
          <a
            href="#home"
            target="_blank"
            class="navi__logoLink"
          >
            <img
              src="./images/logo.png"
              alt="logo"
              class="navi__logoImg"
            >
          </a>
          
          <nav class="navi">
```
add blanks between siblings
it will be more readable